tests/test_visitors/test_ast/test_keywords/test_base_exception.py

```diff
@@ -32,33 +46,29 @@ def test_use_base_exception(
     assert_errors(visitor, [BaseExceptionViolation])
 
 
+@pytest.mark.parametrize('code', [
```

Comment:
You can join `test_exception` and `test_bare_except` together. 
Place both examples into the `parametrize`.